redisinsight/ui/src/components/database-overview/DatabaseOverview.tsx

```diff
@@ -4,14 +4,21 @@ import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiToolTip } from '@elastic/eui'
 
 import { IMetric } from './components/OverviewMetrics'
 
+import AutoRefresh from '../auto-refresh'
 import styles from './styles.module.scss'
 
 interface Props {
   metrics?: Array<IMetric>
+  loadData: () => void
+  lastRefreshTime: number | null
+  handleEnableAutoRefresh: (enableAutoRefresh: boolean, refreshRate: string) => void
 }
 
+const TIMEOUT_TO_GET_INFO = process.env.NODE_ENV !== 'development' ? 5000 : 60_000
```

Comment:
AutoRefresh refreshRate is in seconds

Reply:
Nice catch! Thank you for the clarification. Updated it to also use the environment variable that was added for that